deps/v8/src/isolate.h

```diff
@@ -384,6 +384,11 @@ class Isolate {
  public:
   ~Isolate();
 
+  // ISSUE-14576: allow to access process_wide_mutex_ from sampler.cc
+  static base::LazyMutex GetProcessWideMutex() {
+    return process_wide_mutex_;
```

Comment:
Wouldn't that make a copy of `process_wide_mutex_` every time `GetProcessWideMutex()` is called, and thus `Mutex`'s constructor would be called, which in turn would call `InitializeNativeHandle` and finally `pthread_mutex_init`? If so, is that a problem and should it return a reference instead?